package.json

```diff
@@ -18,19 +18,21 @@
         "multisig": "pnpm --filter ./dapps/multisig-toolkit",
         "changeset-version": "pnpm changeset version && pnpm sdk codegen:version",
         "prettier:check": "prettier -c --ignore-unknown .",
-		"prettier:fix": "prettier -w --ignore-unknown .",
-		"prettier:fix:watch": "onchange '**' -i -f add -f change -j 5 -- prettier -w --ignore-unknown {{file}}",
-		"eslint:check": "eslint --max-warnings=0 .",
-		"eslint:fix": "pnpm run eslint:check --fix",
-		"lint": "pnpm run eslint:check && pnpm run prettier:check",
-		"lint:fix": "pnpm run eslint:fix && pnpm run prettier:fix"
+        "prettier:fix": "prettier -w --ignore-unknown .",
```

Comment:
this whole file should probably be formatted by prettier (.prettierignore currently ignores everything and then re-enables specific directories, this could probably be added there)